client/my-sites/term-selector/no-results.jsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import PureComponent from 'react-pure-render/component';
+import { localize } from 'i18n-calypso';
+
+class TermSelectorNoResults extends PureComponent {
+	constructor( props ) {
+		super( props );
+	}
+
+	render() {
+		const { createLink, translate } = this.props;
+		let createMessage;
+		let noResultsMessage = translate( 'No results. Please try a different search.' );
```

Comment:
This should be `const` since you're not reassigning ([guideline](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#variable-declarations)). I'd probably go one step further and simply inline this in the returned element.